src/Compilers/CSharp/Portable/Binder/Binder_Expressions.cs

```diff
@@ -6053,17 +6053,17 @@ private static void CombineExtensionMethodArguments(BoundExpression receiver, An
                     returnType: returnType);
                 diagnostics.Add(expression, useSiteDiagnostics);
                 var sealedDiagnostics = diagnostics.ToReadOnlyAndFree();
-                var result = new MethodGroupResolution(methodGroup, null, overloadResolutionResult, actualArguments, methodGroup.ResultKind, sealedDiagnostics);
 
-                // If the search in the current scope resulted in any applicable method (regardless of whether a best 
+                // Note: the MethodGroupResolution instance is responsible for freeing its copy of actual arguments
+                var result = new MethodGroupResolution(methodGroup, null, overloadResolutionResult, AnalyzedArguments.GetInstance().Copy(actualArguments), methodGroup.ResultKind, sealedDiagnostics);
```

Comment:
>MethodGroupResolution [](start = 33, length = 21)

:memo: The only three places where we construct a `MethodGroupResolution` with some analyzed arguments are in this file and are visible in this PR.